docs/tutorials/constraints.rst

```diff
@@ -0,0 +1,255 @@
+:orphan:
+
+.. doctest-skip-all
+
+******************************
+Defining Observing Constraints
+******************************
+
+Contents
+========
+
+* :ref:`constraints-built_in_constraints`
+* :ref:`constraints-user_defined_constraints`
+
+.. _constraints-built_in_constraints:
+
+Built-In Constraints
+====================
+
+Frequently, we have a long list of targets that we want to observe, and we need
+to know which ones are observable given a set of constraints imposed on our
+observations by a wide range of limitations. For example, your telescope may
+only point over a limited range of altitudes, your targets are only useful
+in a range of airmasses, and they must be separated from the moon by some
+large angle. The `constraints` module is here to help!
+
+Say we're planning to observe from Subaru Observatory in Hawaii on August 1,
+2015 from 06:00-12:00 UTC. First, let's set up an `astroplan.Observer` object::
+
+    from astroplan import Observer, FixedTarget
+    from astropy.time import Time
+    subaru = Observer.at_site("Subaru")
+    time_range = Time(["2015-08-01 06:00", "2015-08-01 12:00"])
+
+We're keeping a list of targets in a text file called `targets.txt`, which looks
+like this::
+
+    # name ra_degrees dec_degrees
+    Polaris 37.95456067 89.26410897
+    Vega 279.234734787 38.783688956
+    Albireo 292.68033548 27.959680072
+    Algol 47.042218553 40.955646675
+    Rigel 78.634467067 -8.201638365
+    Regulus 152.092962438 11.967208776
+
+We'll read in this list of targets using `astropy.io.ascii`, and create a list
+of `astroplan.FixedTarget` objects out of them::
+
+    # Read in the table of targets
+    from astropy.io import ascii
+    target_table = ascii.read('targets.txt')
```

Comment:
An alternative approach, which *may* be simpler to users (because they've already seen `table` elsewhere in the tutorials) is this:
```
from astropy.table import Table
target_table = Table.read('targets.txt', format='ascii')
```
I'd suggest switching to that just to lower cognitive burden